server/runtime/pom.xml

```diff
@@ -20,6 +20,7 @@
       <defaultExcludedTestGroup />
       <server.output.dir>${project.build.directory}/${infinispan.brand.prefix}-server-${infinispan.brand.version}</server.output.dir>
       <server.test.lib.dir>${project.build.directory}/testlibs</server.test.lib.dir>
+      <version.shrinkwrap.resolvers>3.1.3</version.shrinkwrap.resolvers>
```

Comment:
Should be defined in `build-configuration/pom.xml`

Reply:
Actually, we already have `${org.jboss.shrinkwrap.resolver}` defined in ^ pom and it's on version 2.2.4. Can't we upgrade that to 3.1.3 and reuse it?

We can then just add the following dep in this pom:

```xml
         <dependency>
            <groupId>org.jboss.shrinkwrap.resolver</groupId>
            <artifactId>shrinkwrap-resolver-depchain</artifactId>
            <type>pom</type>
            <scope>test</scope>
         </dependency>
```

Reply:
This is definitely a better approach but I don't know the side effect of changing version for the whole project

Reply:
Let's find out https://github.com/infinispan/infinispan/pull/7458 :slightly_smiling_face:

Reply:
I have folder that PR in here and adjusted other dependencies in the process.